docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -494,15 +494,15 @@ public class FileConfig {
     String format;
 
     /**
-     * The file log level.
+     * The level of logs to be written into the file.
      */
     @ConfigItem(defaultValue = "ALL")
     Level level;
 
     /**
-     * The file logging log level.
+     * The name of file where logs will be stored.
```

Comment:
And we should apply the same change here.

Reply:
Done.

Reply:
same comment here @sarxos

Reply:
@cescoffier This this already modified in accordance to @gsmet request. Please see aab71c4. These two commits will be squashed after I have final approval for these changes.